examples/colab/text_classification_with_tf_hub_on_kaggle.ipynb

```diff
@@ -170,7 +170,7 @@
         "      raise no_file\n",
```

Comment:
Out of place comment:

Could you please try removing the block with "pip installs" and see whether it still works? I think all of the packages should be already running in the colab.

https://github.com/tensorflow/hub/blob/master/examples/colab/text_classification_with_tf_hub_on_kaggle.ipynb?short_path=2e9a14e#L95

Reply:
It works without the pip installs.